### PR TITLE
Sdm post details

### PR DESCRIPTION
**Pull request by seaborndan**
Closes issue #43 and it is now possible to view post details by clicking on the title button in the list.
Epic merge conflict fix
Tested by clicking on aforementioned buttons.

### CODE REVIEW

File: Tabloid/Controllers/PostController.cs [hunk 1]
```diff
@@ -25,6 +25,17 @@ public IActionResult Get()
             return Ok(_postRepository.GetAll());
         }
 
+        [HttpGet("{id}")]
```
Comment (DanielOhrin) [inline Tabloid/Controllers/PostController.cs:28]: Authorize?

**Review by RDollish — APPROVED** [review]
Looks good!

File: Tabloid/Controllers/PostController.cs [hunk 2]
```diff
@@ -25,6 +25,17 @@ public IActionResult Get()
...
+        public IActionResult Post(int id)
```
Comment (jacobgt7) [inline Tabloid/Controllers/PostController.cs:29]: Naming this method "Post" is  a bit confusing, seeing as it's a GET method.  Is there something else you could name it that would indicated what it's doing?

**Review by jsweeten — APPROVED** [review]
Looks good man!

**Review by jacobgt7 — APPROVED** [review]

**Pull request merged**